scripts/run-integration-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+export FASTN_ROOT=`pwd`
+export FASTN_DB_URL=postgres://testuser:testpassword@localhost:5432/testdb
```

Comment:
`export FASTN_DB_URL=${FASTN_DB_URL-postgres://testuser:testpassword@localhost:5432/testdb}` (`-` means if what's on left is not defined, use what's on right, do confirm it somewhere).